nomad/grapl-core.nomad

```diff
@@ -617,21 +634,23 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image        = "${var.container_registry}grapl/graph-merger:${var.graph_merger_tag}"
-        network_mode = "grapl-network"
+        image = "${var.container_registry}grapl/${var.container_repo}graph-merger:${var.graph_merger_tag}"
+      }
+
+      template {
+        data        = local.local_vars
+        destination = "graph-merger.env"
+        env         = true
```

Comment:
so does this actually write stuff to a file named graph-merger.env ?
and then source it as an env?
just as a nomad template baby idk what's going on here tbh

Reply:
Yes, exactly. I do agree that it's weird though

Reply:
Heh, I had this same question... took a little reading to figure out what the heck was going on.